src/screens/internal/CompareSnapshots/utils.ts

```diff
@@ -1,4 +1,4 @@
-import { takeRight as _takeRight } from 'lodash';
+import _takeRight from 'lodash/takeRight';
```

Comment:
why does this work?  Shouldn't it be `takeRight as _takeRight` or whatever?

Reply:
It's the default export from `lodash/takeRight`, we can name it whatever we want. I should probably rename it to `takeRight` though for consistency

Reply:
Oh, weird!  Feel free to leave it.  I thought it was an oversight (that somehow still worked, haha... I probably should have looked closer.)